lib/nexthop_group.c

```diff
@@ -1080,24 +1080,27 @@ void nexthop_group_json_nexthop(json_object *j, const struct nexthop *nh)
 
 	switch (nh->type) {
 	case NEXTHOP_TYPE_IFINDEX:
-		json_object_string_add(j, "nexthop",
+		json_object_string_add(j, "interfaceName",
```

Comment:
this is pretty clearly a fix - it should be a separate commit (and maybe even a separate PR, given the release freeze?)